mkdocs.yml

```
@@ -32,6 +32,7 @@ nav:
       - 'Configure Renovate (Onboarding PR)': 'configure-renovate.md'
       - 'Private npm module support': 'private-modules.md'
       - 'Reconfiguring Renovate': 'reconfigure-renovate.md'
+  - Frequently Asked Questions (FAQ): 'faq.md'
```

```suggestion
  - Frequently Asked Questions: 'faq.md'
```

Do we need the (`FAQ`) part of the title here? It's getting quite long, don't know if that will mess up the styling of the sidebar?